chart/crds/secrets.hashicorp.com_hcpauths.yaml

```diff
@@ -68,11 +68,11 @@ spec:
                   reasons, only project-level service principals should ever be used.
                 properties:
                   secretRef:
-                    description: "SecretRef is the name of a Kubernetes secret in
-                      the consumer's (VDS/VSS/PKI/HCP) namespace which provides the
-                      HCP ServicePrincipal clientID, and clientSecret. \n The secret
-                      data must have the following structure { \"clientID\": \"clientID\",
-                      \"clientSecret\": \"clientSecret\", }"
+                    description: 'SecretRef is the name of a Kubernetes secret in
+                      the consumer''s (VDS/VSS/PKI/HCP) namespace which provides the
```

Comment:
```quote
                      the consumer''s (VDS/VSS/PKI/HCP) namespace which provides the
```

I'm confused, does this work? I see two ticks here, wont that break the string, or remove the punctuation?
Same potential problem in the other file?

Reply:
Not sure, this is generated from `make manifests`

Reply:
The CRDs are always synced to the chart, as well.

Reply:
Seems like it is just escaping the single quote:
```
$ kubectl -n vault-secrets-operator-system get crds hcpauths.secrets.hashicorp.com -o yaml|grep consumer
                      the consumer's (VDS/VSS/PKI/HCP) namespace which provides the
```

Reply:
Wild, alright then :D 👍🏽